JuiceMaker/JuiceMaker/Model/JuiceMaker.swift

```diff
@@ -9,4 +9,20 @@ import Foundation
 // 쥬스 메이커 타입
 struct JuiceMaker {
     
+    let fruitStore = FruitStore(number: 10)
+    
+    func make(_ fruitJuice: FruitJuice) {
+        let ingredient = fruitJuice.ingredient
+        var reducedList: [Fruit: Int] = [:]
+        for (fruit, count) in ingredient {
+            if fruitStore.checkFruit(name: fruit, count: count) {
+                reducedList[fruit] = count
+            } else {
+                print("\(fruitJuice)주스 생성실패")
+                return
+            }
+        }
+        reducedList.forEach{ fruitStore.reduceStock(of: $0.key, count: $0.value) }
```

Comment:
```suggestion
        reducedList.forEach { fruitStore.reduceStock(of: $0.key, count: $0.value) }
```
중괄호의 시작 전, space를 해주는 컨벤션을 많이 따릅니다☺️ 공식문서의 여러 코드를 보시면 확인하실 수 있을거에요!

Reply:
수정했습니다!